crates/augurs-js/src/prophet.rs

```diff
@@ -27,12 +27,12 @@ const OPTIMIZER_FUNCTION: &'static str = r#"
  * `optimizer` export.
  *
  * @param init - The initial parameters for the optimization.
- * @param data - The data for the optimization.
+ * @param data - JSON representation of the data for the optimization.
  * @param opts - The optimization options.
  * @returns An object containing the the optimized parameters and any log
  *          messages.
  */
-type ProphetOptimizerFunction = (init: ProphetInitialParams, data: ProphetStanData, opts: ProphetOptimizeOptions) => ProphetOptimizeOutput;
+type ProphetOptimizerFunction = (init: ProphetInitialParams, data: string, opts: ProphetOptimizeOptions) => ProphetOptimizeOutput;
```

Comment:
TODO: Add a newtype for the `data` arg, add some comments about required fields?